widgetastic_manageiq/__init__.py

```diff
@@ -3437,17 +3437,18 @@ def all_entity_names(self):
         # get_all uses self.entity_names, which handles versioned name query
         return [e.name for e in self.get_all(surf_pages=True)]
 
-    def get_all(self, surf_pages=False):
+    def get_all(self, surf_pages=False, slice=None):
         """ obtains all entities like QuadIcon displayed by view
         Args:
             surf_pages (bool): current page entities if False, all entities otherwise
+            slice(int): if None, return all elements, else return the first {slice} elements.
```

Comment:
Since this `slice` in no longer an `int` this docblock needs updating.

Reply:
updated